tests/utils/fileHostUtils.go

```diff
@@ -0,0 +1,85 @@
+package utils
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+type Catalog []ImageMeta
+
+type ImageMeta struct {
+	Name     string `json:"name"`
+	ItemType string `json:"type"`
+	Mtime    string `json:"mtime"`
+	Size     int32  `json:"size"`
+}
+
+type catalog interface {
+	List() []string
+	Size(string) (int, error)
+}
+
+var _ catalog = Catalog{}
+
+func GetCatalog(ep, accessKey, secretKey string) (*Catalog, error) {
```

Comment:
golint suggests a comment, starting w/ the func name, before all exported funcs.